### ISSUE TITLE
Apple TV: all data, settings everything gets reset frequently. 

### ISSUE THREAD

**Issue by Akakakakakk** [andrefmsilva/iplaytvaio-bugs#147]
**Describe the bug**
As the title says every time the app is closed or the Apple TV is off all the data is cleared. The url, username, password and of course channels, favourites are all wiped out. It’s like it’s newly installed app on the device. This doesn’t happen on the smart tv app and FireStick (both not iplaytv). 

**To Reproduce**

**iOS version**: Version of iOS this bug occurred on (can be found in Settings.app > General)
**iPlayTV AIO version**: Version of iPlayTV this bug occurred on (can be found in Settings tab)
**iOS device**: For example iPhone XS, iPad mini, etc.
**Jailbroken**: Whether or not your device is jailbroken
**Reproducible:** Can you reproduce it every time? Or is it random?

**Expected behavior**

**Screenshots**
If possible, add screenshots to help show the problem.

**Video**
If possible, attach a video to help show the problem. You can attach a video by uploading it to Streamable, Gfycat, YouTube, etc. and pasting the URL here.

**Additional context**

**Comment by tj-leeson** [comment]
Bump for this issue. Occurrence seems to be random although this morning it happened after the playlist file was modified. Not sure if this is coincidence or not.

**Comment by Jumpin2000** [comment]
Agree, this is really random and annoying

**Comment by Heezen2** [comment]
Same issues here.

**Comment by Dribour** [comment]
Same issue. Is there a resolution date ?
Or as a workaround, please add a backup /restore of config.
This happens so often !!!!!

**Comment by ghost** [comment]
agreed. very tedious and annoying, as it takes me 40-50 to customize my channels.
